eclass/distutils-r1.eclass

```diff
@@ -1386,9 +1390,13 @@ distutils_pep517_install() {
 			)
 			;;
 		meson-python)
+			# variables defined by setup_meson_src_configure
+			local MESONARGS=() BOOST_INCLUDEDIR BOOST_LIBRARYDIR
+			setup_meson_src_configure "${DISTUTILS_ARGS[@]}"
```

Comment:
It also sets `NM` and `READELF`. And wasn't the call to `python_export_utf8_locale` supposed to be moved elsewhere?

Reply:
Not sure where the best place to move meson.eclass's python_export_utf8_locale call to is...

Reply:
`meson_src_configure`? distutils-r1 is calling it already.